File: zbar-sample/src/main/java/me/dm7/barcodescanner/zbar/sample/SimpleScannerFragmentActivity.java
package me.dm7.barcodescanner.zbar.sample;

import android.os.Bundle;

import me.dm7.barcodescanner.zbar.sample.databinding.ActivitySimpleScannerFragmentBinding;

public class SimpleScannerFragmentActivity extends BaseScannerActivity {
    @Override
    public void onCreate(Bundle state) {
        super.onCreate(state);
        ActivitySimpleScannerFragmentBinding binding = ActivitySimpleScannerFragmentBinding.inflate(getLayoutInflater());
        setContentView(binding.getRoot());
        setupToolbar(binding.toolbar);
    }
}

File: zbar-sample/src/main/java/me/dm7/barcodescanner/zbar/sample/SimpleScannerFragmentActivity.kt
package me.dm7.barcodescanner.zbar.sample

import android.os.Bundle
import me.dm7.barcodescanner.zbar.sample.databinding.ActivitySimpleScannerFragmentBinding

class SimpleScannerFragmentActivity : BaseScannerActivity() {

    override fun onCreate(state: Bundle?) {
        super.onCreate(state)

        val binding = ActivitySimpleScannerFragmentBinding.inflate(layoutInflater)
        setContentView(binding.root)

        setupToolbar(binding.toolbar)
    }
}
